Introspection/package_tests.py

```diff
@@ -155,9 +166,8 @@ def find_adhoc_bins_libs(self, installed_packages=[]):
             # installed by RPM
             print len(bins)
             bins = list(set(bins) - set(self.get_installed_files(package)))
-            print len(bins)
 
-        return bins
+        return [b for b in bins if not self.match_bin_with_pkg(b)]
```

Comment:
No. I want to list down the files present in the system which are not installed via any RPMs. Method `match_bin_with_pkg` uses `rpm` command line utility. If a file is not installed via any RPM, the method returns `None`, I am filtering the files from all the files in the system return None via the mentioned method.

Finding the metadata is job of another method and are different utilities.